anypointmq/README.md

```diff
@@ -0,0 +1,288 @@
+# Anypoint MQ Bindings
+
+This document defines how to describe Anypoint MQ-specific information in AsyncAPI documents. 
+
+[Anypoint MQ](https://docs.mulesoft.com/mq/) is MuleSoft's multi-tenant, cloud messaging service and is fully integrated with [Anypoint Platform](https://www.mulesoft.com/platform/enterprise-integration).
+
+<a name="version"></a>
+## Versions
+
+The version of this bindings specification is `0.0.1`.
+This is also the `bindingVersion` for all binding objects defined by this specification.
+In any given binding object, `latest` MAY alternatively be used to refer to the currently latest published version of this bindings specification.
+
+## Terminology
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this bindings specification are to be interpreted as described in IETF [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## Protocol
+
+These bindings use the `anypointmq` [protocol](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#definitionsProtocol) in AsyncAPI documents to denote connections to and interactions with Anypoint MQ message brokers.
+
+The Anypoint MQ protocol is based on invocations of the [Anypoint MQ Broker REST API](https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-mq-broker/).
+
+## Server Object
```

Comment:
This entire section is confusing, it mixes AsyncAPI spec with the binding, like it is a part of the Anypoint MQ binding spec. I like how it is done with [IBM MQ](https://github.com/asyncapi/bindings/blob/master/ibmmq/README.md) that treats such info as "considerations", a kind of best practice on how people should use AsyncAPI in combination with this binding.